pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1023,23 +1024,32 @@ func TestAttacherGetDeviceMountPath(t *testing.T) {
 	csiAttacher := getCsiAttacherFromVolumeAttacher(attacher, testWatchTimeout)
 
 	pluginDir := csiAttacher.plugin.host.GetPluginDir(plug.GetPluginName())
-
 	testCases := []struct {
-		testName          string
-		pvName            string
-		expectedMountPath string
-		shouldFail        bool
+		testName      string
+		pvName        string
+		volumeId      string
+		skipCSISource bool
+		shouldFail    bool
+		addVolSource  bool
```

Comment:
is addVolSource used in testcases?

Reply:
yes in https://github.com/kubernetes/kubernetes/pull/107065/files#diff-b1c8c49f572090ada630eecfd3630b04898d467b6cf4c2abba40f053fd606f3aR1063